x-pack/plugins/maps/public/classes/layers/raster_tile_layer/raster_tile_layer.ts

```diff
@@ -53,16 +59,31 @@ export class RasterTileLayer extends AbstractLayer {
   }
 
   async syncData({ startLoading, stopLoading, onLoadError, dataFilters }: DataRequestContext) {
-    const sourceDataRequest = this.getSourceDataRequest();
-    if (sourceDataRequest) {
-      // data is immmutable
-      return;
+    const source = this.getSource();
+    const nextMeta: RasterTileSourceData = {
+      ...dataFilters,
+      url: await source.getUrlTemplate(dataFilters),
```

Comment:
This exposes all `dataFilters` to the source. Is that too broad @nreese? Should we limit it to just `TimeRange` and `timeslice`?

Reply:
I think exposing all filters is good since other custom sources may need access to more then just time time state